.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules/*
 .DS_Store
 .idea
 .iml
+.github/CODEOWNERS
```

Comment:
So in this case, what is the model of updating the codeowners, since changes are ignored? 🤔

Reply:
I am fairly sure tracked files are no longer ignored and this will lead to merge conflicts, I don’t mind testing this out though - if u previously tested this on a fork and didn’t encounter any issues.

Reply:
Tested this, this leads to merge conflicts. https://github.com/mokimo/milo/pull/3

Reply:
@mokimo the ignore [file](https://github.com/mokimo/milo/blob/main/.gitignore) isn't merged yet

Reply:
It is on stage, but even merging it on main is irrelevant - I've included it on main as well and still has the same conflicts.

Reply:
Looks like this setup needs a bit of rework to make this work.